api/src/main/java/org/ehcache/CacheManager.java

```diff
@@ -25,10 +25,38 @@
  */
 public interface CacheManager {
 
+  /**
+   * Creates a {@link Cache} in this {@code CacheManager} according to the specified {@link CacheConfiguration}.
```

Comment:
I believe you do not need the _fqcn_ if the class is in the import list. And I tend to use the _fqcn_ when I want to reference a class that is not in the import list.